xbmc/filesystem/FileCache.cpp

```diff
@@ -164,6 +164,10 @@ bool CFileCache::Open(const CURL& url)
         cacheSize = CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_cacheMemSize;
       }
 
+      // Cap chunk size by cache size
+      if (m_chunkSize > cacheSize)
+        m_chunkSize = cacheSize;
+
```

Comment:
Just out of curiousity. Would the other way round harm? E.g. increasing the cache so that at least m_chunksize fits in it?

Reply:
It will use more memory than required.